hieradata/dev/wso2/wso2is/pattern-3/default.yaml

```diff
@@ -163,53 +167,74 @@ wso2::master_datasources:
    validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
    validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
 
- wso2_is_db:
-   name: WSO2_IS_DB
-   description: The datasource is used for user management and userstore
-   driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
-   url: jdbc:mysql://192.168.48.19:3306/WSO2_IS_DB?autoReconnect=true
-   username: root
-   password: root
-   jndi_config: jdbc/WSO2_IS_DB
-   max_active: "%{hiera('wso2::datasources::common::max_active')}"
-   max_wait: "%{hiera('wso2::datasources::common::max_wait')}"
-   test_on_borrow: "%{hiera('wso2::datasources::common::test_on_borrow')}"
-   default_auto_commit: "%{hiera('wso2::datasources::common::default_auto_commit')}"
-   validation_query: "%{hiera('wso2::datasources::mysql::validation_query')}"
-   validation_interval: "%{hiera('wso2::datasources::common::validation_interval')}"
-
 wso2::bps_datasources:
   bps_ds:
     name: BPS_DS
     description: The datasource used for bps
-    driver_class_name: "%{hiera('wso2::datasources::h2::driver_class_name')}"
-    url: jdbc:h2:file:repository/database/jpadb;DB_CLOSE_ON_EXIT=FALSE;MVCC=TRUE
-    username: "%{hiera('wso2::datasources::common::username')}"
-    password: "%{hiera('wso2::datasources::common::password')}"
+    driver_class_name: "%{hiera('wso2::datasources::mysql::driver_class_name')}"
+    url: jdbc:mysql://192.168.100.1:3306/WSO2_CONFIG_DB?autoReconnect=true
```

Comment:
The datasource name seems to be `bps_ds` while the database name is `WSO2_CONFIG_DB`. Is this correct?